### PR TITLE
cloud: support backup using EBS snapshots

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/backup-and-restore.md [hunk 1]
```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
 
     - In **Backup Retention**, configure the minimum backup data retention period.
 
+    - Turn on or off the **EBS snapshot** feature.
+
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:45]: Question: Do we need to mention this feature is in beta and not recommended for production environments?

Comment (allengaoo) [reply tidb-cloud/backup-and-restore.md:45]: I do not think so.

File: tidb-cloud/backup-and-restore.md [hunk 2]
```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
...
+        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
+        
+        - Minimize or nearly eliminate the impact of backup on cluster performance, for example, keeping the impact on QPS and transaction latency less than 1%, and not consuming any cluster CPU or memory.
+        - Achieve fast backup and restore times, such as completing backup within 1 hour and restore within 2 hours.
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:49]:
```suggestion
        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
        
        - Minimize or nearly eliminate the impact of backup on cluster performance, for example, keeping the impact on QPS and transaction latency below 1%, and not consuming any cluster CPU or memory.
        - Achieve fast backup and restore times, such as completing backup within 1 hour and restore within 2 hours.
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Review by ran-huang — APPROVED** [review]

File: tidb-cloud/backup-and-restore.md [hunk 3]
```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
 
     - In **Backup Retention**, configure the minimum backup data retention period.
 
+    - Turn on or off the **EBS snapshot** feature.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:44]:
```suggestion
    - Turn on or off the **EBS snapshot** feature (beta).
```

File: tidb-cloud/backup-and-restore.md [hunk 4]
```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
...
+
+        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
+        
+        - Minimize or nearly eliminate the impact of backup on cluster performance, for example, keeping the impact on QPS and transaction latency below 1%, and not consuming any cluster CPU or memory.
+        - Achieve fast backup and restore times, such as completing backup within 1 hour and restore within 2 hours.
+
+      To use the EBS snapshot feature, make sure that the following requirements are met:
+
+            - Your TiDB cluster is hosted on AWS and your TiDB version is at least v6.5.0. 
+            - If your TiDB version is v6.5.0, the TiKV node size is at least 4 vCPU and 16 GiB.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:54]:
```suggestion
            - If your TiDB version is v6.5.0, the TiKV node size is at least 4 vCPU and 16 GiB. If your TiDB version is v6.5.1 or later, there is no requirement on the TiKv node size.
```

File: tidb-cloud/backup-and-restore.md [hunk 5]
```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
 
     - In **Backup Retention**, configure the minimum backup data retention period.
 
+    - Turn on or off the **EBS snapshot** feature.
+
+        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
+        
+        - Minimize or nearly eliminate the impact of backup on cluster performance, for example, keeping the impact on QPS and transaction latency below 1%, and not consuming any cluster CPU or memory.
```
Comment (fengou1) [inline tidb-cloud/backup-and-restore.md:48]: "nearly eliminate" a little strange to me, what about"Minimize the impact on cluster performance during backup, with the potential to keep the impact on QPS and transaction latency under 1%, while not utilizing any cluster CPU or memory."?

File: tidb-cloud/backup-and-restore.md [hunk 6]
```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
...
+        - Achieve fast backup and restore times, such as completing backup within 1 hour and restore within 2 hours.
+
+      To use the EBS snapshot feature, make sure that the following requirements are met:
```
Comment (fengou1) [inline tidb-cloud/backup-and-restore.md:51]:
```suggestion
      To use the EBS snapshot feature, make sure that the following prerequisites must be met:
```

File: tidb-cloud/backup-and-restore.md [hunk 7]
```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
...
+
+            - Your TiDB cluster is hosted on AWS and your TiDB version is at least v6.5.0.
```
Comment (fengou1) [inline tidb-cloud/backup-and-restore.md:53]:
```suggestion
            - The TiDB cluster is hosted on AWS and has a minimum TiDB version of v6.5.0. 
```

File: tidb-cloud/backup-and-restore.md (hunk 4 above)
Comment (fengou1) [inline tidb-cloud/backup-and-restore.md]:
```suggestion
            - If the TiDB version is v6.5.0, the TiKV node config must be a minimum of 4 vCPU and 16 GiB.
```

File: tidb-cloud/backup-and-restore.md [hunk 8]
```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
...
+        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
+        
+        - Minimize or nearly eliminate the impact of backup on cluster performance, for example, keeping the impact on QPS and transaction latency below 1%, and not consuming any cluster CPU or memory.
+        - Achieve fast backup and restore times, such as completing backup within 1 hour and restore within 2 hours.
+
+      To use the EBS snapshot feature, make sure that the following requirements are met:
+
+            - Your TiDB cluster is hosted on AWS and your TiDB version is at least v6.5.0. 
+            - If your TiDB version is v6.5.0, the TiKV node size is at least 4 vCPU and 16 GiB. 
+            - When you restore the data backed up by the **EBS snapshot** feature, the number of TiKV and TiFlash nodes in the new cluster is the same as that of the original cluster.
```
Comment (fengou1) [inline tidb-cloud/backup-and-restore.md:55]:
```suggestion
            - When restoring the data was backed up by the **EBS snapshot** feature, the number of TiKV and TiFlash nodes in the new cluster is the same as that of the original cluster.
```

Comment (qiancai) [reply tidb-cloud/backup-and-restore.md:55]: 这里只能是 When you restore，因为当使用 When doing xxx, sb/sth xx 时候的时候，两者的主语必须相同，否则就是垂悬结构语法错误。可参见这里 https://baike.baidu.com/item/%E6%82%AC%E5%9E%82%E7%BB%93%E6%9E%84/3924722?fr=aladdin

File: tidb-cloud/backup-and-restore.md (hunk 7 above)
Comment (qiancai) [reply tidb-cloud/backup-and-restore.md:53]: Your 是和前面的 You can use it to 对应。 At least 为了和 PITR 里的描述尽量一致，at least v6.5.0 我可以改成 v6.5.0 or later.

File: tidb-cloud/backup-and-restore.md [hunk 9]
```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
 
     - In **Backup Retention**, configure the minimum backup data retention period.
 
+    - Turn on or off the **EBS snapshot** feature (beta).
+
+        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
+        
+        - Minimize or nearly eliminate the impact of backup on cluster performance, for example, keeping the impact on QPS and transaction latency below 1%, and not consuming any cluster CPU or memory.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:48]:
```suggestion
        - Minimize the impact of backup on cluster performance, with the potential to keep the impact on QPS and transaction latency under 1% while using no cluster CPU or memory resources.
```

File: tidb-cloud/backup-and-restore.md [hunk 10]
```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
...
+        - Achieve fast backup and restore times, such as completing backup within 1 hour and restore within 2 hours.
+
+      To use the EBS snapshot feature, make sure that the following requirements are met:
+
+            - Your TiDB cluster is hosted on AWS and your TiDB version is at least v6.5.0. 
+            - If your TiDB version is v6.5.0, the TiKV node size is at least 4 vCPU and 16 GiB. If your TiDB version is v6.5.1 or later, there is no requirement on the TiKV node size.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:54]:
```suggestion
```

File: tidb-cloud/backup-and-restore.md [hunk 11]
```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
...
+    - Turn on or off the **EBS snapshot** feature (beta).
+
+        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
+        
+        - Minimize or nearly eliminate the impact of backup on cluster performance, for example, keeping the impact on QPS and transaction latency below 1%, and not consuming any cluster CPU or memory.
+        - Achieve fast backup and restore times, such as completing backup within 1 hour and restore within 2 hours.
+
+      To use the EBS snapshot feature, make sure that the following requirements are met:
+
+            - Your TiDB cluster is hosted on AWS and your TiDB version is at least v6.5.0.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:53]:
```suggestion
            - Your TiDB cluster is hosted on AWS and your TiDB version is v6.5.1 or later
```

File: tidb-cloud/backup-and-restore.md (hunk 6 above)
Comment (qiancai) [reply tidb-cloud/backup-and-restore.md:51]: prerequisites are met 就可以了。 prerequisites 本身就是 something that is necessary before something else can happen or be done

File: tidb-cloud/backup-and-restore.md [hunk 12]
```diff
@@ -49,7 +63,7 @@ By the automatic backup, you can back up the cluster data every day at the backu
         - Resolve cases of data write errors by restoring point-in-time that is before the error event.
         - Audit the historical data of the business.
 
-        If you have one of the preceding needs and want to use the PITR feature, make sure that your TiDB cluster version is at least v6.4.0 and the TiKV node size is at least 8 vCPU and 16 GiB.
+        To use the PITR feature, make sure that your TiDB cluster version is at least v6.4.0, the TiKV node size is at least 8 vCPU and 16 GiB, and the EBS snapshot feature is disabled.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:66]:
```suggestion
        To use the PITR feature, make sure that your TiDB cluster version is v6.4.0 or later, the TiKV node size is at least 8 vCPU and 16 GiB, and the EBS snapshot feature is disabled.
```

File: tidb-cloud/backup-and-restore.md [hunk 13]
```diff
@@ -41,6 +41,19 @@ By the automatic backup, you can back up the cluster data every day at the backu
...
+
+        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
+        
+        - Minimize the impact of backup on cluster performance, with the potential to keep the impact on QPS and transaction latency under 1% while using no cluster CPU or memory resources.
+        - Achieve fast backup and restore times, such as completing backup within 1 hour and restore within 2 hours.
+
+      To use the EBS snapshot feature, make sure that the following prerequisites are met:
+
+            - Your TiDB cluster is hosted on AWS and your TiDB version is v6.5.1 or later.
+            - When you restore the data backed up by the **EBS snapshot** feature, the number of TiKV and TiFlash nodes in the new cluster is the same as that of the original cluster.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:54]:
```suggestion
            - When you restore the data backed up by the **EBS snapshot** feature to a new cluster, the number of TiKV and TiFlash nodes in the new cluster is the same as that of the original cluster.
```

File: tidb-cloud/backup-and-restore.md [hunk 14]
```diff
@@ -41,6 +41,19 @@ By the automatic backup, you can back up the cluster data every day at the backu
...
+        The EBS snapshot feature supports backing up and restoring data using EBS snapshots. You can use it to:
+        
+        - Minimize the impact of backup on cluster performance, with the potential to keep the impact on QPS and transaction latency under 1% while using no cluster CPU or memory resources.
+        - Achieve fast backup and restore times, such as completing backup within 1 hour and restore within 2 hours.
+
+      To use the EBS snapshot feature, make sure that the following prerequisites are met:
+
+            - Your TiDB cluster is hosted on AWS and your TiDB version is v6.5.1 or later.
+            - When you restore the data backed up by the **EBS snapshot** feature to a new cluster, the number of TiKV and TiFlash nodes in the new cluster is the same as that of the original cluster.
+            - The EBS snapshot feature and PITR feature are mutually exclusive. Therefore, only one of them can be enabled at a time.
```
Comment (qiancai) [inline tidb-cloud/backup-and-restore.md:55]:
```suggestion
      - Your TiDB cluster is hosted on AWS and your TiDB version is v6.5.1 or later.
       - When you restore the data backed up by the **EBS snapshot** feature to a new cluster, the number of TiKV and TiFlash nodes in the new cluster is the same as that of the original cluster.
       - The EBS snapshot feature and PITR feature are mutually exclusive. Therefore, only one of them can be enabled at a time.
```

**Comment by qiancai** [comment]
/type/enhancement
/needs-cherry-pick-release-4.0
/needs-cherry-pick-release-5.0
/needs-cherry-pick-release-5.1
/needs-cherry-pick-release-5.2
/needs-cherry-pick-release-5.3
/needs-cherry-pick-release-5.4
/area/tidb-lightning
/needs-cherry-pick-release-6.1
/needs-cherry-pick-release-6.5
/needs-cherry-pick-release-6.6
/needs-cherry-pick-release-7.0
/needs-cherry-pick-release-7.1

**Comment by qiancai** [comment]
Close this PR because this feature is not in the plan anymore.

**Pull request closed without merging**